config/locales/devise.en.yml

```diff
@@ -3,9 +3,9 @@
 en:
   devise:
     confirmations:
-      confirmed: "Your email address has been successfully confirmed."
-      send_instructions: "You will receive an email with instructions for how to confirm your email address in a few minutes."
-      send_paranoid_instructions: "If your email address exists in our database, you will receive an email with instructions for how to confirm your email address in a few minutes."
+      confirmed: "Your account is now confirmed! Let's start looking at opportunities."
+      send_instructions: "We sent you an email with a link to confirm your account. It might take you a few minutes to receive it."
```

Comment:
If we're going to modify this translation, let's go ahead and remove the "few minutes".